### DISCUSSION TITLE
installing @toruslabs/react-native-web-browser makes app crash

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#525] [Help]
Im trying to implement web3auth in react native the weird thing is the app crashes just after installing this dependency 
* What went wrong:
Execution failed for task ':toruslabs_react-native-web-browser:compileDebugKotlin'.
> Compilation error. See log for more details

**Comment by shahbaz17** [comment]
Duplicate to https://github.com/orgs/Web3Auth/discussions/529
